homeassistant/components/delijn/sensor.py

```diff
@@ -83,20 +83,23 @@ async def async_update(self):
         self._attributes["stopname"] = self._name
 
         try:
-            first = self.line.passages[0]
-            if first["due_at_realtime"] is not None:
-                first_passage = first["due_at_realtime"]
+            if len(self.line.passages) > 1: """do a check if there is a passage"""
```

Comment:
Why do we require 2 items? Is that intended?

Reply:
You're correct, forgot that we count from '0' :-). One item is enough.
Will correct this.